File: SwordDemoLibrary/utilclass/src/main/java/com/example/utilclass/LogUtil.java
package com.example.utilclass;

import android.os.Environment;
import android.util.Log;

import java.io.File;

public class LogUtil {
    private static final String TAG = "Sword"; 
    private static final String LOG_SWITCH_DIR = Environment.getExternalStorageDirectory().getAbsolutePath() + File.separator + "LoggerSwitch";
    
    public LogUtil() {
        String switchFileName = Encryption.md5(LOG_SWITCH_DIR + File.separator + "logger_switch");
        
    }
    
    private static boolean isDebug() {
        
    }
    
    public static void debug(String msg) {
        Log.d(TAG, msg);
    }
    
    public static void error(String msg) {
        Log.d(TAG, msg);
    }
    
    public static void warn(String msg) {
        Log.d(TAG, msg);
    }
}

File: SwordDemoLibrary/connectivity/src/main/java/com/example/connectivity/util/LogUtil.kt
@file:JvmName("LogUtilKt")
package com.example.connectivity.util
import android.util.Log


class LogUtil {
    object INSTANCE {
        private val TAG = "connectivity"

        fun debug(str: String) {
            Log.d(TAG, str)
        }
    }


}
